src/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import './Footer.css';
+
+function App() {
+  return (
+    <footer className="footer">
+    <div className="upper_container">
+        <div className="footer-col">
+
+        <img className="img-logo"src="images/Logo.svg" alt="" style="color: white;"/>
+
+        </div>
+        <div className="footer-col">
+            
+                <li><a href="#">About us</a></li>
+                <li><a href="#">International tickets</a></li>
+                <li><a href="#">Terms and Conditions</a></li>
+
+        </div>
+        <div className="footer-col">
+            
+                <li><a href="#">Strategy and vision</a></li>
+                <li><a href="#">Integrated Transportation Systems</a></li>
+            
+        </div>
+        <div className="footer-col">
+            <div>
+                <script src="https://code.iconify.design/iconify-icon/1.0.1/iconify-icon.min.js"></script>
+                <iconify-icon class= icons icon="cib:rss" style= "color: white"></iconify-icon>
```

Comment:
instead of this element use https://mui.com/material-ui/material-icons/. 
or add support for this tag because now we get error on npm start 
![image](https://user-images.githubusercontent.com/81109647/201607516-9c947651-2c02-4c00-acce-e60c8b5b0672.png)